roles/deploy/tasks/main.yml

```diff
@@ -142,6 +142,7 @@
   args:
     chdir: "{{ releases_path }}"
   become: yes
+  tags: skip_ansible_lint
```

Comment:
The lint writer would probably be happier about this version:
```patch
diff --git a/roles/deploy/tasks/main.yml b/roles/deploy/tasks/main.yml
index e4d7d15..f59ac77 100644
--- a/roles/deploy/tasks/main.yml
+++ b/roles/deploy/tasks/main.yml
@@ -126,6 +126,7 @@
 
 - name: create a database backup version
   command: cp {{ rollback_sql_path }} {{ releases_path }}/{{ timestamp }}.sql.gz
+  register: backup_creation
   tags: skip_ansible_lint
   when: table_exists.stderr.find('does not exist') == -1
 
@@ -135,7 +136,7 @@
 
 - name: create a repo backup version
   command: cp -r {{ rollback_path }} {{ releases_path }}/{{ timestamp }} removes={{ rollback_path }}
-  tags: skip_ansible_lint
+  when: backup_creation is succeeded
 
 - name: delete old release files
   shell: set -o pipefail && ls -1tr | head -n -10 | xargs -d '\n' rm -rf --
```

It connects the two steps. There are many lint skips in this file though. The nature of this deploy script doesn't fit well into the Ansible paradigm. I think most of these commands would actually be better in a shell script on the server (which is installed and executed by Ansible). That's out of scope of this PR though.

I don't mind if you leave this as is.